agent/engine/engine_integ_test.go

```diff
@@ -295,20 +295,16 @@ func TestContainerHealthCheck(t *testing.T) {
 
 	go taskEngine.AddTask(testTask)
 
-	event := <-stateChangeEvents
-	assert.Equal(t, event.(api.ContainerStateChange).Status, api.ContainerRunning, "Expected container to be RUNNING")
-
-	event = <-stateChangeEvents
-	assert.Equal(t, event.(api.TaskStateChange).Status, api.TaskRunning, "Expected task to be RUNNING")
+	verifyTaskIsRunning(stateChangeEvents, testTask)
```

Comment:
Let's not use `verifyTaskIsRunning` anymore since it's slated for deprecation. Please use https://github.com/aws/amazon-ecs-agent/blob/1b9439dda781fd8aa4b5f6e5750b23774f7a930a/agent/engine/engine_integ_test.go#L100 and https://github.com/aws/amazon-ecs-agent/blob/1b9439dda781fd8aa4b5f6e5750b23774f7a930a/agent/engine/engine_integ_test.go#L107 instead

Same goes for `verifyTaskIsStopped` later in the test.